config/features.yml

```diff
@@ -1155,9 +1155,6 @@ features:
   my_va_lighthouse_uploads_report:
     actor_type: user
     description: Use lighthouse /uploads/report endpoint for Form status
-  my_va_form_submission_statuses:
```

Comment:
SubmissionStatusesController, the corresponding spec file, and the swagger spec file still references `my_va_form_submission_statuses`

Reply:
Flipper still being called in [swagger specs](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/requests/swagger_spec.rb#L3751).